modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -78,7 +78,24 @@ above will no longer run in IE11 and will fail to bootstrap with an error and/or
 
 ### React 18 Upgrade
 
-> Coming Soon (https://github.com/Workday/canvas-kit/discussions/1545)
+We've upgraded Canvas Kit to [React 18](https://reactjs.org/blog/2022/03/29/react-v18.html). React
+18 introduces several new features, but the most impactful change is the new
+[concurrent render API](https://reactjs.org/blog/2022/03/29/react-v18.html#what-is-concurrent-react).
+The new `createRoot` method replaces `render` and gives you access to the improvements in React 18
+including the concurrent features. Without it, React behaves as it did in version 17.
+
+Canvas Kit doesn’t use the render API directly in our packages but Storybook uses it to render
+Canvas Kit components. Because of this, we upgraded Storybook dependencies to support React 18.
+
+This change will not impact product teams that are already on v7 and are looking to upgrade to v8 as
+v8 will support the same React versions as v7 while still giving teams the ability to upgrade to
+React 18 as needed. Canvas Kit will continue to support React 16.14, React 17.X in v8, so it allows
+to upgrade to Canvas Kit v8 without upgrading to React 18. But some new features may be ignored and
```

Comment:
> But some new features may be ignored and
limited by the legacy root API usage and a console warning about `.root()` being deprecated will be
presented.

I'm also wondering what features could be ignored and I couldn't find more info on a `.root()` console warning. My understanding is that if users are on React 16.14+ or 17, they will build Canvas Kit successfully without any console warnings or ignored features.